Subject: [PATCH] delete before replace EC2 instance to detach Volume before creating new EC2 instance and attach Volume to it
---
Index: index.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.ts b/index.ts
--- a/index.ts	(revision 6f125c040f26717dc2a44aa687646149b7688785)
+++ b/index.ts	(date 1693486752747)
@@ -88,7 +88,7 @@
   },
   {
     // necessary since volumeDetachment doesn't seem to respect stopInstanceBeforeDetaching
-    // deleteBeforeReplace: true
+    deleteBeforeReplace: true
   },
 )
 
