Subject: [PATCH] delete before replace EC2 instance to detach Volume before creating new EC2 instance and attach Volume to it
---
Index: index.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.ts b/index.ts
--- a/index.ts	(revision 65e3b052798b1179c4563f70e57eaf8169ee1e01)
+++ b/index.ts	(date 1693485137837)
@@ -85,7 +85,7 @@
   },
 }, {
   // necessary since volumeDetachment doesn't seem to respect stopInstanceBeforeDetaching
-  // deleteBeforeReplace: true
+  deleteBeforeReplace: true
 });
 
 const webDataEbs = new aws.ebs.Volume("web", {
